src/App.js

```diff
@@ -1,12 +1,19 @@
+import { useEffect } from 'react';
```

Comment:
Hi @Gegardus, 

Good job. Congratulations. You can merge now. Optional: Please fix design to be same like in given design.




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._